db/seeds.rb

```diff
@@ -16,22 +16,150 @@
+# Recommendation.create(emotion: Emotion.find_by(name: "anxious"), action: Action.find(2))
+# Recommendation.create(emotion: Emotion.find_by(name: "disgust"), action: Action.find(5))
+# Recommendation.create(emotion: Emotion.find_by(name: "stress"), action: Action.find(7))
+
+Emote.create(input: "I am happy", recommendation: Recommendation.find_by(emotion: Emotion.find_by(name: "joy")))
+
+
+
+actions = {
+	stress: [
+		"Stretch.", 
+		"Take a break to play a short game.", 
+		"Go for a walk.", 
+		"Doodle.", 
+		"Write down your lingering thoughts.", 
+		"Focus on structured breathing.", 
+		"Call a close friend.", 
+		"Immerse yourself in a hobby you enjoy.", 
+		"Work out.", 
+		"Listen to soothing music.", 
+		"Listen to nature sounds.", 
+		"Go for a run.", 
+		"Take a short nap.", 
+		"Go for a bike ride.", 
+		"Medidate.", 
+		"Go for a walk.", 
+		"Have a conversation with friends or family.", 
+		"Go outside.", 
+		"Have a glass of water.", 
+		"Play with your pet, or visit an animal shelter.", 
+		"Picture a relaxing scene.", 
+		"Sit in the sun for 15 minutes."
+	],
+	anger: [
+		"Listen to your favorite song.", 
+		"Take a moment to be present in the moment.", 
+		"Write a letter you do not intend to send.", 
+		"Take a time out.", 
+		"Go for a walk.", 
+		"Take deep breaths and think about something that makes you happy.", 
+		"Go for a bike ride.", 
+		"Go for a run.", 
+		"Talk with friends or family.", 
+		"Get some fresh air.", 
+		"Go to the gym.", 
+		"Go to the movies."
+	],
+	fear: [
+		"Text a friend.", 
+		"Focus on the present moment.", 
+		"Focus on your breathing.", 
+		"Seek out support.", 
+		"Go for a walk.", 
+		"Watch a comforting movie.", 
+		"Meditate.", 
+		"Exercise.", 
+		"Talk to someone you trust.", 
+		"Change your surroundings.", 
+		"Listen to music.", 
+		"Take a deep breath.", 
+		"Share your feelings with a close friend."
+	],
+	joy: [
+		"Go for a walk.", 
+		"Call a friend.", 
+		"Make someone laugh.", 
+		"Hug a loved one.", 
+		"Exercise.", 
+		"Smile at a stranger.", 
+		"Remain in the present.", 
+		"Go outside.",
+		"Mow your lawn."
+	],
+	sadness: [
+		"Do some push-ups.", 
+		"Spend some time with your pet, or visit an animal shelter.", 
+		"Seek out support.", 
+		"Write down your thoughts.", 
+		"Take time to get out of the house.", 
+		"Breathe.", 
+		"Spend some time outside.", 
+		"Take a short nap.", 
+		"Drink a cup of tea.", 
+		"Watch a funny movie.", 
+		"Talk to family or friends.", 
+		"Focus your enery on work or a hobby.", 
+		"Spend some time to take care of an easy task."
+	],
+	anxious: [
+		"Eat a burrito.",
+		"Focus on breathing.", 
+		"Have a cup of tea.", 
+		"Speak with someone you trust.", 
+		"Seek out a friend or loved one.", 
+		"Go for a walk.", 
+		"Treat yourself to a snack that you enjoy.", 
+		"Meditate.", 
+		"Take a deep breath.", 
+		"Stretch.", 
+		"Go for a run.", 
+		"Listen to relaxing music.", 
+		"Make a list."
```

Comment:
Delete lines 122, 123, 130, 131

Add - 
"Inhale as deeply as you can, and exhale slowly.",
  "Drink a cold glass of water or milk.",
  "Interact with your pet, or visit an animal shelter.",
  "Take a moment to stretch.",
  "Take stock of your immediate environment, and change one thing for the better."